src/com/wsscode/pathom/diplomat/http.cljc

```diff
@@ -3,14 +3,42 @@
 
 (s/def ::encode-type (s/with-gen keyword? #(s/gen #{::transit+json ::edn ::json})))
 
-(s/def ::url string?)
+(s/def ::url (s/or :string string? :uri uri?))
```

Comment:
`uri?` is clojure only, for compatibility reasons I think we better keep this to strings